pages/common/ssh-keygen.md

```diff
@@ -33,3 +29,7 @@
 - Change the password of a key:
 
 `ssh-keygen -p -f ~/.ssh/{{filename}}`
+
+- Change the type of the key format (for example from OPENSSH format to PEM), the file will be rewritten in-place:
+
+`ssh-keygen -p -N "" -m PEM -f ~/.ssh/{{filename}}`
```

Comment:
```suggestion
`ssh-keygen -p -N "" -m {{PEM}} -f ~/.ssh/{{filename}}`
```
Where is the `OPENSSH` part here? If the description mentions it, but it's nowhere in the example, it might be confusing

Reply:
OPENSSH is the format of the private key , like somebody explained it here https://coolaj86.com/articles/the-openssh-private-key-format/

Reply:
```suggestion
`ssh-keygen -p -N "" -m {{PEM -f {{~/.ssh/OpenSSH_private_key}}`
```

Reply:
Keys can be stored in other places if wanted, and the keys can be chosen in the config file using `IdentityFile /path/to/key`